pkgs/applications/science/biology/whisper/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "whisper";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "refresh-bio";
+    repo = "Whisper";
```

Here you can also use 
```suggestion
    repo = pname;
```